### PR TITLE
feat(redirect-url): Fixed MDN fetch.

### PR DESCRIPTION
**Pull request by hemanth**
__From:__

image

__To:__

image

### CODE REVIEW

File: src/plugins/mdn/mdnPlugin.js [hunk 1]
```diff
@@ -30,11 +30,14 @@ function extractFromHtml(html) {
...
     .text();
 
+  console.log({text})
   if (!text) {
     const bodyText = $('body')
       .text()
       .replace(/\s+/g, ' ');
 
+    console.log($('body').text())
+    console.log({bodyText})
```
Comment (ljharb) [inline src/plugins/mdn/mdnPlugin.js:40]: these should probably be removed?

File: src/plugins/mdn/mdnPlugin.js [hunk 2]
```diff
@@ -72,14 +72,14 @@ async function fixLanguage(origRes, lastRedirect) {
...
-  const match = script.match(reg);
+  const meta = $('meta[http-equiv="refresh"]').attr("content")
+  const reg = /url=\/l\/\?uddg\=(.*)/
+  const match = meta.match(reg);
   if (!match) {
     return res;
   }
 
-  const redirect = decodeURIComponent(match[2]);
+  const redirect = decodeURIComponent(match[1].replace('/l/?uddg=', ''));
```
Comment (brigand) [inline src/plugins/mdn/mdnPlugin.js:82]: I don't think this .replace does anything

Comment (hemanth) [reply src/plugins/mdn/mdnPlugin.js:82]: Oh yeah, that was from before line-76, will remove that.

File: src/plugins/mdn/mdnPlugin.js [hunk 3]
```diff
@@ -72,14 +72,14 @@ async function fixLanguage(origRes, lastRedirect) {
 
 async function fixRedirect(res) {
   const $ = cheerio.load(res.text);
-  const script = $('script').get()[0].children[0].data;
-  const reg = /(window.location.replace\('\/l\/\?kh=-1&uddg=)(.*)(\'\))/;
-  const match = script.match(reg);
+  const meta = $('meta[http-equiv="refresh"]').attr("content")
+  const reg = /url=\/l\/\?uddg\=(.*)/
```
Comment (brigand) [inline src/plugins/mdn/mdnPlugin.js:76]: The `=` doesn't need to be escaped

Comment (hemanth) [reply src/plugins/mdn/mdnPlugin.js:76]: Nods, but any side-effect?

**Comment by brigand** [comment]
Thanks!

**Pull request merged**